03-task-manager/starter/app.js

```diff
@@ -5,20 +5,22 @@ const app = express()
 const tasks = require('./routes/tasks')
 const connectDB = require('./db/connect')
 require('dotenv').config()
-
+const notFound = require('./middleware/not-found')
+const errorHandlerMiddleware = require('./middleware/error-handler')
 
 //middleware
+app.use(express.static('./public'))
 app.use(express.json())
 
 
 //routes
-app.get('/hello', (req,res) => {
-    res.send('Task Manager App')
-})
 
 app.use('/api/v1/tasks',tasks)
 
-const port = 3000 
+app.use(notFound)
+app.use(errorHandlerMiddleware)
+
+const port = process.env.PORT || 3000
 
 const start = async () => {
     try { 
```

Comment:
This is not completed, finish the try/catch block and make sure to start() the server